packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -38,27 +40,114 @@ const GlobalClassNames = {
   custom: 'ms-Panel--custom'
 };
 
-const panelSize = {
-  width: {
-    xs: '272px',
-    sm: '340px',
-    md: '643px',
-    lg: '940px'
+const panelWidth = {
+  full: '100%',
+  auto: 'auto',
+  xs: 272,
+  sm: 340,
+  md1: 592,
+  md2: 644,
+  lg: 940
+};
+
+const panelMargin = {
+  auto: 'auto',
+  none: 0,
+  md: 48,
+  lg: 428,
+  xl: 176
+};
+
+const smallPanelSelectors = {
```

Comment:
Comment explaining how/where these are used might be good (it's fairly obvious from reading the code, but having it in words is even better)

Reply:
Fixing